packages/api/cms-api/src/mailer/mailer.module.ts

```diff
@@ -10,6 +12,8 @@ export type MailerModuleConfig = {
     defaultFrom: string;
     sendAllMailsTo?: string[];
     sendAllMailsBcc?: string[];
+    disableMailLog?: boolean;
+    daysToKeepMailLog?: number; // default: 90
```

Comment:
We could specify the default value using JSDoc:

```ts
/**
 * @default 90
 * /
```

Then devs could see it in the IDE. If we don't want this, then I'd remove the comment.